veecle-orchestrator/src/conductor.rs

```diff
@@ -298,6 +339,27 @@ impl Conductor {
             })
             .await;
     }
+
+    /// Stops and removes all runtime instances.
+    #[tracing::instrument(skip(self))]
+    pub(crate) async fn clear(&self) {
+        self.shutdown().await;
+
+        let ipc_tasks = Vec::from_iter(self.runtimes.lock().unwrap().drain().filter_map(
```

Comment:
Do you need to create the Vec here ?
I think you can just call `.for_each(|id, mut runtime| {...})` on the iterator